app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -63,10 +75,33 @@ public void sendOrder(final LabOrder<?> order) throws UnableToSendLabOrderExcept
         logger.logInfo("Sending the order to ReportStream at {}", RS_DOMAIN_NAME);
 
         String json = fhir.encodeResourceToJson(order.getUnderlyingOrder());
-        String bearerToken = requestToken();
+        String bearerToken;
+        bearerToken = getRsToken();
```

Comment:
Is there a reason to have the instantiation in a separate line?

Reply:
changed.